frontend/site-registry/src/app/features/details/disclosure/Disclosure.tsx

```diff
@@ -11,50 +11,52 @@ import './Disclosure.css';
-        dateRegistrar:new Date('2013-05-31'),
-        dateEntered:new Date('2013-05-31'),
-        disclosureSchedule:[
-            {
-                scheduleId:1,
-                reference:'F1',
-                discription:'PETROLEUM OR NATURAL GAS DRILLING',
-                sr:true
-            },
-            {
-                scheduleId:2,
-                reference:'F2',
-                discription:'PETROLEUM OR NATURAL GAS PRODUCTION FACILITIES',
-                sr:false,
-            },
-        ],
-        summary: 'PLANNED ACTIVITIES INCLUDE MEETING THE OBLIGATIONS OF THE ENVIRONMENTAL MANAGEMENT ACT AND CONTAMINATED SITES REGULATION TO OBTAIN A CERTIFICATE OF RESTORATION FOR THE PROPERTY. THE END LAND USE OF THE PROPERTY IS WILDLANDS - REVERTED.',
-        statement:`SITE DISCLOSURE WAS COMPLETED AND SUMMARIZED USING AVAILABLE SITE INFORMATION OBTAINED VIA A FILE REVIEW OF WELLSITE DOCUMENTS OBTAINED FROM ENERPLUS CORPORATION'S CALGARY OFFICE. ADDITIONAL SITE BACKGROUND INFORMATION OBTAINED FROM USING A REVIEW OF AVAILABLE HISTORICAL AERIAL PHOTOGRAPHS AND A SEARCH OF ON-LINE DATABASES MAINTAINED AND/OR DEVELOPED BY REGULATORY AGENCIES (OIL AND GAS COMMISSION AND MINISTRY OF THE ENVIRONMENT).`,
-        governmentOrder:'NONE.',
-        srTimeStamp: `Sent to SR on ${formatDate(new Date())}`,
-};
+
+// const disclosureData = {
+//         disclosureId:1,
+//         siteId:1,
+//         dateReceived:new Date('2013-05-31'),
+//         dateComplete:new Date('2013-05-31'),
+//         localAuthorityReceived:new Date('2013-05-31'),
+//         dateRegistrar:new Date('2013-05-31'),
+//         dateEntered:new Date('2013-05-31'),
+//         disclosureSchedule:[
+//             {
+//                 scheduleId:1,
+//                 reference:'F1',
+//                 discription:'PETROLEUM OR NATURAL GAS DRILLING',
+//                 sr:true
+//             },
+//             {
+//                 scheduleId:2,
+//                 reference:'F2',
+//                 discription:'PETROLEUM OR NATURAL GAS PRODUCTION FACILITIES',
+//                 sr:false,
+//             },
+//         ],
+//         summary: 'PLANNED ACTIVITIES INCLUDE MEETING THE OBLIGATIONS OF THE ENVIRONMENTAL MANAGEMENT ACT AND CONTAMINATED SITES REGULATION TO OBTAIN A CERTIFICATE OF RESTORATION FOR THE PROPERTY. THE END LAND USE OF THE PROPERTY IS WILDLANDS - REVERTED.',
+//         statement:`SITE DISCLOSURE WAS COMPLETED AND SUMMARIZED USING AVAILABLE SITE INFORMATION OBTAINED VIA A FILE REVIEW OF WELLSITE DOCUMENTS OBTAINED FROM ENERPLUS CORPORATION'S CALGARY OFFICE. ADDITIONAL SITE BACKGROUND INFORMATION OBTAINED FROM USING A REVIEW OF AVAILABLE HISTORICAL AERIAL PHOTOGRAPHS AND A SEARCH OF ON-LINE DATABASES MAINTAINED AND/OR DEVELOPED BY REGULATORY AGENCIES (OIL AND GAS COMMISSION AND MINISTRY OF THE ENVIRONMENT).`,
+//         governmentOrder:'NONE.',
+//         srTimeStamp: `Sent to SR on ${formatDate(new Date())}`,
+// };
```

Comment:
pls remove the commented code

Reply:
Keeping it for future reference because table value is not confirmed